README.md

```diff
@@ -4,20 +4,29 @@ You can deploy 1Password SCIM Bridge on any supported infrastructure that allows
 
 ## Automatic (one-click) deployment
 
-The easiest way to deploy 1Password SCIM Bridge is with our one-click apps, which are currently available for Google Cloud Platform and DigitalOcean:
+For a guided deployment, you can use our Google Cloud or DigitalOcean marketplace app to deploy 1Password SCIM Bridge:
 
 - [Deploy 1Password SCIM Bridge on Google Cloud Platform](https://support.1password.com/scim-deploy-gcp/)
 - [Deploy 1Password SCIM Bridge on DigitalOcean](https://support.1password.com/scim-deploy-digitalocean/)
 
-## Advanced deployment
+## Custom deployment
 
-If you have particular requirements for your environment, an advanced deployment is recommended. These example configurations can be used as a base for a customized deployment.
+For a more customizabile deployment, whether you choose a container-as-a-service option to allow the infrastructure to be managed by your service provider, or an advanced deployment, you can use these example configurations as a base for a customized deployment.
```

Comment:
To simplify this a bit for improved readability:

```suggestion
To customize your deployment, you can use containers as a service or create your own advanced deployment with the examples below as your base.
```

We explain CaaS above so we don't need to again here, similar to how we're not defining what "advanced deployment" means.